server.js

```diff
@@ -98,3 +197,9 @@ if (!isProduction) {
 		);
   });
 }
+
+function emailAuthorized(email) {
```

Comment:
As per code review session, this should be moved as an exported function in /server/auth

One thing I forgot to mention, it should return a boolean and not throw so when called:

```
if (!isEmailAuthorized(user.email) {
  res.redirect(401, '/error');
}
next();
```

I believe in Node, when you throw, the server cannot handle passing status codes and rendering view layer templates.